### DISCUSSION TITLE
Is there a way to easily list all tags?

### DISCUSSION THREAD

**Discussion by claman** [ttscoff/doing#176] [Q&A]
I sometimes forget which tags I'm using for which project(s), and scrolling back through entries feels less than ideal. I'm trying to figure out if there's a way to list all unique tags currently in the doc. Is there a built-in way to do this that I just haven't encountered? Or should I try to write a separate script?

**Comment by ttscoff** [comment]
`doing tags`?

-Brett

**Reply by claman** [reply]
...and this what I get for trying to read before drinking coffee. Sorry to bother you with the inane question!
